src/enforcer/HolographERC721.sol

```diff
@@ -901,6 +949,21 @@ contract HolographERC721 is Admin, Owner, HolographERC721Interface, Initializabl
     }
   }
 
+  function _sourceCall(bytes memory payload) private returns (bool output) {
```

Comment:
What is this function protecting?

Reply:
Since we use msgSender function in place of msg.sender for source contract interactions between holographer, there are cases where we made calls without that being enabled/present. This creates an issue where custom contracts can make the assumption that the msgSender will always return data which was not the case. This change in code fixes the issue.